userspace/libscap/CMakeLists.txt

```diff
@@ -139,3 +139,10 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 
 	include(FindMakedev)
 endif()
+
+option(CREATE_TEST_TARGETS "Enable make-targets for unit testing" ON)
```

Comment:
sorry for my lack of cmake knowledge but do we need to redefine this option even if we have already defined it in the root `CMakeLists.txt`? I've seen that we do the same thing in `libsinsp` `CMakeLists.txt` ... :thinking:

Reply:
I kept this for consistency with the rest, could perhaps be useful when only some libraries need to be compiled?